permissions/src/androidLibMain/kotlin/Permissions.kt

```diff
@@ -30,11 +31,11 @@ actual class Permissions {
     internal lateinit var context: Context
 
     actual fun getBluetoothManager(): PermissionManager {
-        return BluetoothPermissionManager(context)
+        return BluetoothPermissionManager()
     }
 
     actual open class Builder {
-        private lateinit var context: Context
+        private var context: Context = ContextProvider.context
```

Comment:
As suggested in the issue.. it's probably the most decent thing to still allow passing a context in the constructor, but use ContextProvider if no argument is passes.

Advantages:
- `val` > `var`
- No crashes if you don't want to use ~~ContextHolder~~ ApplicationContextHolder
- No need to pass context every time you build either